src/components/checkbox/index.tsx

```diff
@@ -225,6 +234,10 @@ function createStyles(props: CheckboxPropTypes) {
       tintColor: iconColor,
       alignItems: 'center',
       justifyContent: 'center'
+    },
+    checkboxLabel: {
+      marginLeft: Spacing.s3,
```

Comment:
Assuming you'll except the suggestion at the top comment, then..
```suggestion
      marginLeft: Spacings.s3,
```